### ISSUE TITLE
Emoji!

### ISSUE THREAD

**Issue by MegaApuTurkUltra** [OpenSprites/OpenSprites-next#48]
We need some emoji 📦 

I propose combining Scratch's forum emojis with their comment emojis and 📦  and :shipit: 

What emojis should we have?

**Comment by jamesd-uk** [comment]
kappa?

@nanalan

**Comment by bates64** [comment]
Emoji should be instanced with `_emoji_` syntax, that Scratch uses, or the `:emoji:` syntax that GH uses.
Also :tada: needs to be an emoji too

**Comment by bates64** [comment]
> I propose combining **Scratch's forum emojis**

...we should find a nice-looking set; like the Google ones.

**Comment by Tymewalk** [comment]
@nanalan [EmojiOne](http://emojione.com) looks nice. Similar to Twitter emojis and they're licensed under Creative Commons.

**Comment by MegaApuTurkUltra** [comment]
~~We could just use emojify.js~~ 📦  jk let's use EmojiOne, seems better

**Comment by andrewjcole** [comment]
If you're looking for original ones, I'd be willing to make some.

**Comment by Tymewalk** [comment]
@andrewjcole Maybe you could do them in that style you did for the "leaving OpenSprites" page?

**Comment by bates64** [comment]
@andrewjcole @Tymewalk it might look a little weird considering the small size of emoji and that doodle style - I say we use EmojiOne

**Comment by Tymewalk** [comment]
@nanalan You make a good point, although we should still consider it. Original emoji would look nicer than just using EmojiOne.

**Comment by andrewjcole** [comment]
I agree with @nanalan, that style probably wouldn't really work too well in such a small size. I do like that EmojiOne has SVG support, but it is gigantic, and we have no idea what kind of emojis are in there. I'll leave it to you guys to decide.

**Comment by Tymewalk** [comment]
@andrewjcole EmojiOne has EVERY emoji. As in, all the emojis Unicode has ever added.
We could just pick out a few from the entire set and use those.

**Comment by Arinerron** [comment]
Someone should work on this cause it is "World Emoji Day" (who even decided that?!)

**Comment by bates64** [comment]
So I guess the general is both `_scratch syntax_` and/or `:emoji syntax:` and we'll support basic faces like :stuck_out_tongue_winking_eye: :smile: :cry: and :angry:, and lastly ones like :package: :shipit: :+1: and :-1:?

**Comment by andrewjcole** [comment]
This probably isn't urgent for release?
